packages/instant/src/redux/actions.ts

```diff
@@ -25,7 +25,11 @@ function createAction<T extends string, P>(type: T, data?: P): PlainAction<T> |
 export enum ActionTypes {
     UPDATE_ETH_USD_PRICE = 'UPDATE_ETH_USD_PRICE',
     UPDATE_SELECTED_ASSET_AMOUNT = 'UPDATE_SELECTED_ASSET_AMOUNT',
-    UPDATE_BUY_ORDER_STATE = 'UPDATE_BUY_ORDER_STATE',
+    SET_BUY_ORDER_STATE_NONE = 'SET_BUY_ORDER_STATE_NONE',
```

Comment:
I know we spoke about the pattern of `UPDATE`-prefixed actions taking a parameter and `SET`-prefixed actions not taking any parameters, but `UPDATE_BUY_ORDER_STATE_PROCESSING` reads a lot less naturally than `SET_BUY_ORDER_STATE_PROCESSING` IMHO.  Also, I like having the same prefix for all actions that have to do with buy order state, even if some take parameters and others don't.

Reply:
Don't feel strongly about this. The reasoning was that `UPDATE_BUY_ORDER_STATE_PROCESSING` would never really be a name used -- rather it would be `UPDATE_BUY_ORDER_STATE` or `SET_BUY_ORDER_STATE_PROCESSING`